checkout.html

```diff
@@ -0,0 +1,111 @@
+
+<!doctype html>
+<html lang="en"><head>
+  <meta charset="utf-8">
+  <title>Checkout</title>
+  <link rel="stylesheet" href="styles/stylesheet.css">
+  <link href="https://fonts.googleapis.com/css2?family=Neuton:wght@200&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div>
+    <header>
+      <img src="images/adoptadog-logo.svg" alt="Smiley face" height="35%" width="35%">
+      <img class = "imageclass"  src="images/cart-icon.svg" alt="Smiley face">  <span class = "headertextspan">$123.456</span>
+      <button class="button-checkout"> <a class = "abutton" href="checkout.html"> Checkout </a></button>
+
+      <p class="headerp"> Seattle's premier dog adoption agency </p>
+    </header>
+
+  <nav class="menu">
+    <ul class="navlist">
+      <li><a href="index.html">Home  </a></li>
+      <li><a href="dogs.html">Dogs</a></li>
+      <li><a href="blog.html">Blog</a></li>
+    </ul>
+  </nav>
+  <div>
+    <section class = "sectionblog">
+      <h1 class="dogsh1"> Checkout </h1>
+      <div class="clearfix-form">
+
+        <form action="/action_page.php">
+          <label class = "label">Name</label>
+          <input type="text" id="fname" name="firstname" placeholder="Your name..">
+
+          <label class = "label">Email Address</label>
+          <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+
+          <label class = "label">Street Address</label>
+          <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+          <div class = "form-div">
+            <label class = "label">City</label>
+            <select id="country" name="country">
+              <option value="australia">Australia</option>
+              <option value="canada">Canada</option>
+              <option value="usa">USA</option>
+            </select>
+
+          <!-- </div>
+          <div class = "form-div"> -->
+            <label class = "label" style = "padding-left: 10px;">State</label>
```

Comment:
Try not to use inline styles in html. Best practice nowadays is to apply all styles inside the stylesheet. Also px are not a responsive unit. Outside of defining border widths, we recommend using %, em, rem units moving forward.